src/engine_render.c

```diff
@@ -473,6 +473,23 @@ void rotate_base_axis(struct M33 *matx, short angle, unsigned char axis)
     matx->r[2].v[3] = matx->r[2].v[0] * matx->r[2].v[1];
 }
 
+struct WibbleTable *get_wibble_from_table(long table_index, MapSubtlCoord stl_x, MapSubtlCoord stl_y)
+{
+    struct WibbleTable *wibl;
```

Comment:
To make WIBBLE=LIQUIDONLY work correctly: I had to add 2 additional slabs to check if we should add wibble to the current subtile or not (`slb2` and `slb3`). 

If I only check the current subtile (stl_x and stl_y) then the DIRT/PATH/ROCK (etc) on the Southern and Western edges of (e.g.) a pool of lava will have wibble.

Maybe there is a better method to fix this issue, but if so it eludes me at the moment.